tests/integration/conftest.py

```diff
@@ -1244,6 +1244,28 @@ def prepared_principal_acl(runtime_ctx, env_or_skip, make_mounted_location, make
     )
 
 
+@pytest.fixture
+def prepared_legacy_acl(runtime_ctx, make_mounted_location, make_catalog, make_schema):
```

Comment:
why do you need to introduce new fixture if it's used only once? it hinders readability

Reply:
I'm anticipating more tests with legacy ACLs. But will remove for now.